src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/common/helper/QueryParamsHelper.java

```diff
@@ -5,21 +5,21 @@
 
 public class QueryParamsHelper {
```

Comment:
class can be declared as final class, only one single private constructor

Reply:
Out of scope